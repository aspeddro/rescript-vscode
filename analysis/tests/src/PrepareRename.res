let a = 1
//  ^pre

let b = 2 and c = 3
//                 ^pre      