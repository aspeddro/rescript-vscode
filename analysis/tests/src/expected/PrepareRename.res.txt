PrepareRename src/PrepareRename.res 0:4
{"start": {"line": 0, "character": 4}, "end": {"line": 0, "character": 5}}

PrepareRename src/PrepareRename.res 3:19
{"start": {"line": 3, "character": 18}, "end": {"line": 3, "character": 19}}

